sys/include/net/ng_ipv6/addr.h

```diff
@@ -47,6 +47,15 @@ extern "C" {
                                             "ffff:ffff:255.255.255.255"))
 
 /**
+ * @brief The first 10 bits of a site-local IPv6 unicast address
+ *
+ * @see <a href="http://tools.ietf.org/html/rfc4291#section-2.5.7">
+ *          RFC 4291, section 2.5.7
+ *      </a>
+ */
+#define NG_IPV6_ADDR_SITE_LOCAL_PREFIX  (0xFEC0)
```

Comment:
I introduced this mainly to decide if an address is global unicast. Since the first 128 bits can be anything for global unicast, the only way to determine this is to exclude all other types.